src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.PackageValidation.targets

```diff
@@ -0,0 +1,40 @@
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. -->
+<Project>
+  <UsingTask TaskName="Microsoft.DotNet.PackageValidation.ValidatePackage" AssemblyFile="$(DotNetPackageValidationAssembly)" />
+
+  <ItemGroup Condition="'$(EnablePackageValidation)' == 'true' and '$(DisablePackageBaselineValidation)' != 'true' and '$(PackageValidationBaselinePath)' == '' and '$(PackageValidationBaselineVersion)' != ''">
+    <PackageDownload Include="$([MSBuild]::ValueOrDefault('$(PackageValidationBaselineName)', '$(PackageId)'))" Version="[$(PackageValidationBaselineVersion)]" />
+  </ItemGroup>
+
+  <Target Name="RunPackageValidation"
+          AfterTargets="Pack"
+          Condition="'$(IsPackable)' == 'true' and '$(EnablePackageValidation)' == 'true'">
+
+    <PropertyGroup Condition="'$(RoslynDirectory)' == ''">
+      <RoslynDirectory>$([System.IO.Path]::GetDirectoryName($(CSharpCoreTargetsPath)))</RoslynDirectory>
```

Comment:
Can you double check that `CSharpCoreTargetsPath` is defined for VB, and F# projects?